src/Support/SimplePaginationType.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rebing\GraphQL\Support;
+
+use GraphQL\Type\Definition\ObjectType;
+use GraphQL\Type\Definition\Type as GraphQLType;
+use Illuminate\Pagination\Paginator;
+use Illuminate\Support\Collection;
+use Rebing\GraphQL\Support\Facades\GraphQL;
+
+class SimplePaginationType extends ObjectType
+{
+    public function __construct(string $typeName, string $customName = null)
+    {
+        $name = $customName ?: $typeName.'SimplePagination';
+
+        $config = [
+            'name'   => $name,
+            'fields' => $this->getPaginationFields($typeName),
+        ];
+
+        $underlyingType = GraphQL::type($typeName);
+        if (isset($underlyingType->config['model'])) {
+            $config['model'] = $underlyingType->config['model'];
+        }
+
+        parent::__construct($config);
+    }
+
+    /**
+     * @param string $typeName
+     *
+     * @return array<string, array<string,mixed>>
+     */
+    protected function getPaginationFields(string $typeName): array
+    {
+        return [
+            'data'           => [
+                'type'        => GraphQLType::listOf(GraphQL::type($typeName)),
+                'description' => 'List of items on the current page',
+                'resolve'     => function (Paginator $data): Collection {
+                    return $data->getCollection();
+                },
+            ],
+            'per_page'       => [
+                'type'        => GraphQLType::nonNull(GraphQLType::int()),
+                'description' => 'Number of items returned per page',
+                'resolve'     => function (Paginator $data): int {
+                    return $data->perPage();
+                },
+                'selectable'  => false,
+            ],
+            'current_page'   => [
+                'type'        => GraphQLType::nonNull(GraphQLType::int()),
+                'description' => 'Current page of the cursor',
+                'resolve'     => function (Paginator $data): int {
+                    return $data->currentPage();
+                },
+                'selectable'  => false,
+            ],
+            'from'           => [
+                'type'        => GraphQLType::int(),
+                'description' => 'Number of the first item returned',
+                'resolve'     => function (Paginator $data): int {
```

Comment:
Nullable field:
```suggestion
                'resolve'     => function (Paginator $data): ?int {
```

Reply:
I got this error from [https://github.com/rebing/graphql-laravel/runs/2265106812](https://github.com/rebing/graphql-laravel/runs/2265106812). Same with "to" field.
`Anonymous function never returns null so it can be removed from the  return typehint.   `